bears/stylus/StylintBear.py

```diff
@@ -0,0 +1,41 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='stylint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):?(?P<column>\d+)?\s+.*'
+                     r'(?P<severity>error|warning)\s+(?P<message>.+)')
+class StylintBear:
+    """
+    Attempts to catch little mistakes (duplication of rules for instance) and
+    enforces a code style guide on Stylus (a dynamic stylesheet language
+    with the `.styl` extension that is compiled into CSS) files.
+
+    The `StylintBear` is able to catch following problems:
```

Comment:
I'm still unsure about using single backticks with rst, could you use double backticks to be safe? Or you can keep using them if you proof me that code highlighting works fine with those too ;)